mini_pupper_driver/setup.py

```diff
@@ -20,13 +20,13 @@
     author_email='fae@mangdang.net',
     maintainer='MangDang',
     maintainer_email='fae@mangdang.net',
-    description='The mini_pupper_control package',
+    description='The mini_pupper_driver package',
     license='MIT',
```

Comment:
```suggestion
    license='Apache-2.0',
```

Could you fix this in this pull reuquest?